src/components/Pages/Admin/Designer.vue

```diff
@@ -362,6 +367,48 @@ export default {
             );
         },
 
+        // Send selected toolbar and perspectives to first student in selected group/ groups.
+        async sendToFirstInGroup(groups) {
+            if (!groups.length) {
+                this.showToast('Please select group to send toolbar for first student', 'warning');
+            } else {
+                const information = this.prepareMetaInformationPerspectivesAndToolbars();
```

Comment:
It would be cleaner to use destructuring here and refer the properties directly later on:
```javascript
const { perspectives, toolbar } = this.prepare /* snip */;
```